platform-includes/configuration/before-send/native.mdx

```diff
@@ -28,7 +38,7 @@ static sentry_value_t
 crash_cleanup(
     const sentry_ucontext_t *uctx, // provides the user-space context of the crash
     sentry_value_t event,          // used the same way as in `before_send`
-    void *closure                  // user-data that you can provide at configuration time
+    void *user_data                  // user-data that you can provide at configuration time
```

Comment:
I'm not quite sure what kind of cleanup one could do with the data one gets from the `uctx` and `event`; @supervacuus do you have any inspiration? I was thinking about using fields from `uctx->siginfo` to discard specific types of crashes, but maybe you know of something nicer.

Reply:
Updated the wording in the [last commit](https://github.com/getsentry/sentry-docs/pull/13984/commits/de9309f920caeb054cfa0e4c40afcaa60ee2e671#diff-f0e10746d49e60c000ac2dfad2b50ae00f64bf45d370fdc105a8fc84b7fe4404R42-R53) , let me know if this better describes the use-case for `on_crash`.